pkg/controlplane/instance.go

```diff
@@ -567,6 +568,22 @@ func labelAPIServerHeartbeatFunc(identity string) lease.ProcessLeaseFunc {
 
 		// convenience label to easily map a lease object to a specific apiserver
 		lease.Labels[apiv1.LabelHostname] = hostname
+
+		// Include apiserver network location used by peers to proxy requests between kube-apiservers
+		if PeerProxyFeatureEnabled() {
+			var ip string
+			if peerBindAddress.PeerBindIP != "" && peerBindAddress.PeerBindPort != "" {
+				ip = fmt.Sprintf("%s.%s", peerBindAddress.PeerBindIP, peerBindAddress.PeerBindPort)
```

Comment:
this format is confusing IP 1.1.1.1 port 80 = 1.1.1.1.80 
IP 2001:db8::2 port 90 2001:db8::2:90

I think is better to use the standard format for urls given by `net.JoinHostPort` that is IPv4:port or [IPv6]:port

Reply:
Done